src/Config/ConfigModule.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
```

Comment:
Could we rename this to EnvModule perhaps? I find the naming confusing. Also could we move it to inside `/src/configs`. Trying to avoid having a lot of clutter in the `/src/` folder. I think long term it will make the app easier to navigate. Thoughts?